src/main/java/mate/jdbc/dao/ManufacturerService.java

```diff
@@ -4,7 +4,7 @@
 import java.util.Optional;
 import mate.jdbc.model.Manufacturer;
 
-public interface ManufacturerDao {
+public interface ManufacturerService {
```

Comment:
Please return back original naming. 2 ManufacturerServices in neighbour packages creates mess